### PR TITLE
BZ-2142763 Prometheus File

### PR DESCRIPTION
**Pull request by OdedViner**
https://bugzilla.redhat.com/show_bug.cgi?id=2142763

### CODE REVIEW

File: tests/manage/monitoring/prometheus/test_deployment_status.py [hunk 1]
```diff
@@ -167,6 +168,16 @@ def test_ceph_osd_stopped(measure_stop_ceph_osd):
         )
 
 
+@bugzilla("2142763")
+def test_prometheus_file():
```
Comment (fbalak) [inline tests/manage/monitoring/prometheus/test_deployment_status.py:172]: This test should be in a different file. You can create a new one in the same directory. Something like `test_prometheus_rules.py`.

File: tests/manage/monitoring/prometheus/test_deployment_status.py [hunk 2]
```diff
@@ -167,6 +168,16 @@ def test_ceph_osd_stopped(measure_stop_ceph_osd):
         )
 
 
+@bugzilla("2142763")
```
Comment (fbalak) [inline tests/manage/monitoring/prometheus/test_deployment_status.py:171]: please add polarion id

**Review by fbalak — CHANGES_REQUESTED** [review]
lgtm but the test should be in a different file

File: tests/manage/monitoring/prometheus/test_deployment_status.py (hunk 2 above)
Comment (OdedViner) [reply tests/manage/monitoring/prometheus/test_deployment_status.py]: done

**Review by fbalak — APPROVED** [review]

**Review by PrasadDesala — APPROVED** [review]

**Pull request merged**